README.md

```diff
@@ -18,8 +18,8 @@ In most cases, running `./install.sh` should be sufficient to install all depend
 - `--env-name=NAME`: Create conda environment named `NAME` instead of the default (ceesd).
 - `--install-prefix=DIR`: Install mirgecom and git pip packages to `DIR` instead of the default (./).
 - `--branch=NAME`: Install the `NAME`d branch of mirgecom instead of the default branch (master).
-- `--conda-pkgs=FILE`: Install additional conda packages from list specified in `FILE`.
-- `--pip-pkgs=FILE`: Install additional pip packages from list specified in `FILE`.
+- `--conda-pkgs=FILE`: Install additional conda packages from the list of package names specified in `FILE`.
```

Comment:
I'd suggest that we make this an [environment file](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#sharing-an-environment), in preparation for conda reproducibility, rather than defining our own, semi-equivalent format.

Reply:
Agreed. Until then, should we merge this patch as it is?

Reply:
Sure.

Reply:
#90